chronograf/1.10/alpine/entrypoint.sh

```diff
@@ -9,4 +9,8 @@ if [ "$1" = 'chronograf' ]; then
   export BOLT_PATH=${BOLT_PATH:-/var/lib/chronograf/chronograf-v1.db}
 fi
 
-exec "$@"
+if [ "$(id -u)" -ne 0 ] || { [ ! -z "${CHRONOGRAF_AS_ROOT}" ] && [ "${CHRONOGRAF_AS_ROOT}" != "false" ]; }; then
```

Comment:
I think this would be simpler and achieve the same result:
```
if [ "$(id -u)" -ne 0 ] || [ "${CHRONOGRAF_AS_ROOT}" != "false" ]; then ...
```

(since you're quoting `${CHRONOGRAF_AS_ROOT}` if it is empty it will evaluate to `""` which does not equal `"false"`)